client/src/stores/user/reducers.js

```diff
@@ -0,0 +1,28 @@
+import { LOAD_USER_DATA, UPDATE_USER_DATA } from "./actions";
+
+const initialState = {
+  attending: [],
+  contactInfo: "",
+  coordinates: {
+    lat: null,
+    lon: null,
+  },
```

Comment:
Let's couple lat and lon together a bit more. We should never have a situation with lat but not lon.

```suggestion
  coordinates: null,
```